content/talks/2017-05-11-Bughanuta for Hanutas.md

```diff
@@ -1,17 +1,16 @@
 ---
-title: SHA1 Break
+title: Bughanuta for Hanutas
```

Comment:
Yeah, It is. But it's also not my talk and that is the title provided by the speaker.